### PR TITLE
feat: Improve tls checker docs and variables

### PR DESCRIPTION
**Pull request by diegolagospagopa**
### List of changes

- improve documentation
- fix health check ports
- changed variables name to be more simple to understand how to use it

### Motivation and context

- During migration to v3 the modules was broken and need a fix

### Type of changes

- [ ] Add new module
- [x] Update existing module
- [ ] Remove existing module

### Does this introduce a breaking change?

- [x] Yes
- [ ] No

### Run checks

Useful commands to run checks on local machine

```sh
bash .utils/terraform_run_all.sh init local
pre-commit run -a
```

### CODE REVIEW

File: tls_checker/templates/tls-cert.yaml.tpl [hunk 1]
```diff
@@ -12,11 +12,11 @@ service:
 
 readinessProbe:
   httpGet:
-    port: 8080
+    port: 80
```
Comment (fmlisco) [inline tls_checker/templates/tls-cert.yaml.tpl:15]: Why this change of port?

**Review by fmlisco — CHANGES_REQUESTED** [review]

**Review by fmlisco — CHANGES_REQUESTED** [review]

File: api_management/README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ resource "azurerm_private_dns_a_record" "api_internal" {
 
 | Name | Version |
 |------|---------|
-| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | 3.53.0 |
+| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | 3.44.1 |
```
[the remark this answers is not in the document]
Comment (diegolagospagopa) [reply api_management/README.md:136]: Yes is correct

File: tls_checker/templates/tls-cert.yaml.tpl (hunk 1 above)
Comment (diegolagospagopa) [reply tls_checker/templates/tls-cert.yaml.tpl:15]: because the previos 8080 was wrong and don't allow the module to work correctly

File: tls_checker/main.tf [hunk 3]
```diff
@@ -20,9 +20,9 @@ resource "helm_release" "helm_this" {
         region                         = var.location_string
         expiration_delta_in_days       = var.expiration_delta_in_days
         host                           = var.https_endpoint
-        appinsights_instrumentationkey = var.application_insights_connection_string
         keyvault_name                  = var.keyvault_name
-        keyvault_tenantid              = var.keyvault_tenantid
+        keyvault_tenant_id              = var.keyvault_tenant_id
```
[the remark this answers is not in the document]
Comment (diegolagospagopa) [reply tls_checker/main.tf]: fixed
